apps/omg_status/lib/status.ex

```diff
@@ -23,7 +23,7 @@ defmodule OMG.Status do
   # if ETS table has a tuple entry in form of {:boot_in_progress, 0}, return false
   # if ETS table has a tuple entry in form of {:boot_in_progress, 1}, return true
   @health_match List.flatten(
-                  for n <- [:boot_in_progress, :ethereum_client_connection],
+                  for n <- [:boot_in_progress, :ethereum_client_connection, :chain_crash],
```

Comment:
`:chain_crash` isn't too clear. I'd also associate a total `crash` with sth leading BEAM to exiting, which is not the case here.

Also (and more importantly) this alarm pertains to both watcher and child chain, whereas `chain` is quite generic and abstract. Even if the watcher is in `chain_crash`, one shouldn't think that the chain crashed or stopped since it is might be just fine (both the `child chain` and the `root chain`.

Alternatives: `main_supervisor_exited`, `ethereum_supervisor_exited/halted`, `monitor_exited`. I think we can be more specific here.